scripts/zones/Rolanberry_Fields/mobs/Simurgh.lua

```diff
@@ -2,11 +2,35 @@
 -- Area: Rolanberry Fields (110)
 --  HNM: Simurgh
 -----------------------------------
-mixins = { require('scripts/mixins/rage') }
+mixins =
+{
+    require('scripts/mixins/rage'),
+    require("scripts/mixins/job_special")
+}
 -----------------------------------
 ---@type TMobEntity
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:addMod(xi.mod.SLEEPRES, 100)
+    mob:setMod(xi.mod.EVA, 400)
```

Comment:
horizon recently had a PR that moved this evasion mod out to onMobSpawn because apparently setMod for evasion doesn't work in onMobInitialize. I guess stats are re-calculated after that is called? not sure. Maybe the next spawn has it wrong and the first one is fine

additionally, we use `mob:addImmunity(xi.immunity.SLEEP)` for immunities now iirc

I'm not a mob/combat guy typically so I will defer to others if i'm wrong

Reply:
Adjusting this immunity has shown me that the blanket sleep immunity has been removed. Also adjusting this for Eldritch Edge to put the immunities in lua instead of mob_pools.

Reply:
Oh yeah, it's light sleep or dark sleep (separated) now